.jenkins/common_job_properties.groovy

```diff
@@ -157,6 +162,7 @@ class common_job_properties {
 
   // Sets common config for PostCommit jobs.
   static def setPostCommit(def context,
+                           def trigger_phrase = '',
```

Comment:
We seem to be mixing styles of `abc_def` and `abcDef`. Check what's default groovy style and unify in a separate PR.